uiowa/uiowa
# [Commencement] Simplify Commencement program archive

# How to test
1. Sync the commencement site and then navigate to [the programs archive page](https://commencement.uiowa.ddev.site/programs#Archive)
2. Observe that each session is linked to a program and that its only one column.
3. Note that you no longer have the ability to sort.
